CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
+- Fixed issue adding a custom network via panel. ([#6965](https://github.com/brave/brave-ios/issues/6965))
+- Fixed issue with ERC721 NFTs with different token identifiers showing incorrect balance. ([#6954](https://github.com/brave/brave-ios/issues/6954))
+- Fixed crash related to loading URLs when selecting one of the on-ramp purchase options. ([#6993](https://github.com/brave/brave-ios/issues/6993))
+
+General UI
+
+- Added support for only loading HTTP/HTTPS URLs when scanning a QR code from the URL bar. ([#6997](https://github.com/brave/brave-ios/issues/6997))
+- Added capability to permanently delete sync accounts. ([#6820](https://github.com/brave/brave-ios/issues/6829))
+- Added promotional callout for VPN cross-platform linking on account.brave.com. ([#6164](https://github.com/brave/brave-ios/issues/6164))
+- Added support for selecting full page view when taking screenshots in Brave. ([#2806](https://github.com/brave/brave-ios/issues/2806))
+- Dropped support for iOS 14 devices. ([#6695](https://github.com/brave/brave-ios/issues/6695))
+- Fixed website favicons flickering when searching history. ([#7116](https://github.com/brave/brave-ios/issues/7116))
+- Fixed issue where some playlist folders were not shown when using CarPlay. ([#7105](https://github.com/brave/brave-ios/issues/7105))
+- Fixed Brave News feed not showing when following only individual sources and no channels. ([#7093](https://github.com/brave/brave-ios/issues/7093))
+- Fixed Brave News feed being updated every time NTP is viewed with RSS-only sources enabled. ([#7059](https://github.com/brave/brave-ios/issues/7059))
+- Fixed suppressing "open in app" dialog on google maps website. ([#7024](https://github.com/brave/brave-ios/issues/7024))
+- Fixed an issue with cross-platform VPN configuration not being refreshed after credentials expire. ([#7014](https://github.com/brave/brave-ios/issues/7014))
+- Fixed issue where full URL is shown under the shields panel. ([#6964](https://github.com/brave/brave-ios/issues/6964))
+- Fixed Playlist folder reordering feature. ([#6961](https://github.com/brave/brave-ios/issues/6961))
+- Fixed "Blocking Switch to App Notices" switch from being in disabled state causing "open in app" dialogs to not be suppressed on webpages. ([#6953](https://github.com/brave/brave-ios/issues/6953))
+- Fixed favorites not showing in the widget. ([#6859](https://github.com/brave/brave-ios/issues/6859))
+- Fixed opening ad notifications that use items-appss:// where new tab is opened but isn’t resolved. ([#6782](https://github.com/brave/brave-ios/issues/6782))
+- Fixed an issue where scrolling/swiping stops on "www.theblock.co" after loading links using "Open in New Tab" option. ([#6518](https://github.com/brave/brave-ios/issues/6518))
+- Fixed Shields panel toggle alignment for localized German string. ([#6171](https://github.com/brave/brave-ios/issues/6171))
```

Comment:
nit: `Shields` doesn't need to be capitalized as you're referring to the panel and not the "feature". Would be different if we were using lets say `Brave Shields` which is mentioning the feature.